pkg/services/notificator/deps.go

```diff
@@ -0,0 +1,19 @@
+package notificator
+
+import objectSDKAddress "github.com/nspcc-dev/neofs-sdk-go/object/address"
+
+// NotificationSource provides notification that should
```

Comment:
Just
```suggestion
// NotificationSource is a source of object notifications.
```
Interface should not be related to other interface `NotificationWriter`.